### PR TITLE
Update link to akka vavr sample project

### LINKED ISSUE #26269: Broken links to samples
It seems that a lot of links to sample projects are broken in the documentation, f.e. these pages all contain broken links:

https://doc.akka.io/docs/akka/current/multi-node-testing.html
https://doc.akka.io/docs/akka/current/camel.html
https://doc.akka.io/docs/akka/current/actors.html (link to vavr project)
https://doc.akka.io/docs/akka/current/persistence.html (link to source code)

### PR DESCRIPTION
**Pull request by Philippus**
## Purpose

Fixes a broken link. Not sure if this is the best solution, but I couldn't find the project on this page: https://developer.lightbend.com/start/?group=akka.

## References

Fixes #26269

### CODE REVIEW

**Comment by johanandren** [comment]
I don't think that sample ever was on developer.lightbend.com so that is probably a too eager re-link at some point, maybe we should make it get published there though.

**Comment by raboof** [comment]
I'd prefer fixing this by publishing the sample there, yes ;)

**Comment by patriknw** [comment]
That sample hasn't been updated for Akka 2.6 yet, and I'm not sure we should spend time on converting it to Typed. Is vavr popular? I think it could be good enough to link to the 2.5 branch of the sample.

File: akka-docs/src/main/paradox/actors.md [hunk 1]
```diff
@@ -832,7 +832,7 @@ That has benefits such as:
 The `Receive` can be implemented in other ways than using the `ReceiveBuilder` since it in the
 end is just a wrapper around a Scala `PartialFunction`. In Java, you can implement `PartialFunction` by
 extending `AbstractPartialFunction`. For example, one could implement an adapter
-to [Vavr Pattern Matching DSL](http://www.vavr.io/vavr-docs/#_pattern_matching). See the @extref[Akka Vavr sample project](samples:akka-sample-vavr) for more details.
+to [Vavr Pattern Matching DSL](http://www.vavr.io/vavr-docs/#_pattern_matching). See the [Akka Vavr sample project](https://github.com/akka/akka-samples/tree/2.6/akka-sample-vavr) for more details.
```
Comment (patriknw) [inline akka-docs/src/main/paradox/actors.md]: even link to the 2.5 branch since we might remove the sample for 2.6

**Review by johanandren — APPROVED** [review]
LGTM, thanks @Philippus

**Review by raboof — APPROVED** [review]

**Comment by patriknw** [comment]
OK TO TEST

**Pull request merged**